src/mmap.cpp

```diff
@@ -230,10 +268,12 @@ file_handle::file_handle(string_view name, std::int64_t const size
 		// https://docs.microsoft.com/en-us/windows/desktop/api/fileapi/nf-fileapi-setfilevaliddata
 		std::call_once(g_once_flag, acquire_manage_volume_privs);
 
+#ifndef TORRENT_WINRT
```

Comment:
you should probably move this `#ifndef` above the call to `acquire_manage_volume_privs` too, since that's only there to support `SetFileValidData()`